CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 ### 0.46.5
 - Add an imperative reset handler to reset `PinInputField` from the parent component with ref.
+- Adds a `focusFirstInputOnReset` prop for the above change, to opt out of focusing on first input on reset.
```

Comment:
Change “Adds” to “Add”, keeping in line with the rules :P 
And remove the full-stops at the end, please?